schema.sql

```diff
@@ -0,0 +1,12 @@
+/* Database schema to keep the structure of entire database. */
+
+CREATE TABLE animals (
+    id INT,
```

Comment:
- [ ] [OPTIONAL] Kindly make the id column not null and autoincremented, so you don't have to insert it manually.
```
id int NOT NULL IDENTITY(1,1)
```

Reply:
Thank you for the suggestion, I will make the change right now